cli/src/cmd/testnet/destroy.rs

```diff
@@ -19,19 +20,16 @@ impl TestnetDestroyCmd {
     /// Execute the destroy command
     pub fn run(&self, home_dir: &Path) -> Result<()> {
         if !home_dir.exists() {
-            println!(
-                "⚠️  Testnet directory does not exist at {}",
-                home_dir.display()
-            );
+            warn!("Testnet directory does not exist at {}", home_dir.display());
             return Ok(());
         }
 
         // Confirm with user unless --force is specified
         if !self.force {
-            println!("⚠️  This will stop all nodes and permanently delete all testnet data at:");
-            println!("   {}", home_dir.display());
-            println!();
-            print!("   Are you sure? (y/N): ");
+            warn!("This will stop all nodes and permanently delete all testnet data at:");
+            info!("   {}", home_dir.display());
```

Comment:
The level change here is a bit odd, I think a 1 liner as warning would be better